shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/network/MealApi.kt

```diff
@@ -62,6 +62,14 @@ class MealApi {
         }
     }
 
+    suspend fun reserveAMeal(id: String): Meal {
+        var meal = getMeal(id)
+        if (meal.quantity > 0) {
```

Comment:
i think there should be a kotlin equivalent to isEmpty in Swift, i think they even have isNotEmpty()